tools/sloop.c

```diff
@@ -0,0 +1,84 @@
+/**
+ * Library of functions pertaining to Sloop shadows. Slooping is a way
+ * to simulate the built-in snooping functionality of EotL.
+ *
+ * @author gon
+ * 2017-05
+ */
+
+inherit AnsiTellCode;
```

Comment:
definitely not. I made a brief attempt to do my ansi_tells from sloop.c and quickly abondoned it, so I'll get those removed.